pennylane/qnode_new/qnode.py

```diff
@@ -30,41 +29,6 @@
 from pennylane.qnode import QNode as QNode_old, QuantumFunctionError, decompose_queue
 
 
-def qnode(device, *, mutable=True, properties=None):
-    """Decorator for creating QNodes.
-
-    When applied to a quantum function, this decorator converts it into
-    a (wrapped) :class:`QNode` instance.
-
-    Args:
-        device (~.Device): a PennyLane-compatible device
-        mutable (bool): whether the node is mutable
-        properties (dict[str->Any]): additional keyword properties passed to the QNode
-    """
-
-    @lru_cache()
-    def qfunc_decorator(func):
-        """The actual decorator"""
-
-        node = QNode(func, device, mutable=mutable, properties=properties)
-
-        @wraps(func)
-        def wrapper(*args, **kwargs):
-            """Wrapper function"""
-            return node(*args, **kwargs)
-
-        # bind the jacobian method to the wrapped function
-        # wrapper.jacobian = node.jacobian
-        wrapper.metric_tensor = node.metric_tensor
-
-        # bind the node attributes to the wrapped function
-        wrapper.__dict__.update(node.__dict__)
-
-        return wrapper
-
-    return qfunc_decorator
-
-
 _MARKER = inspect.Parameter.empty  # singleton marker, could be any singleton class
```

Comment:
By the way, what is the point of this line? `_MARKER` is not used anywhere

Reply:
@smite?